app/static/app/potree/workers/BinaryDecoderWorker.js

```diff
@@ -270,7 +270,7 @@ Potree.Version.prototype.equalOrHigher = function(version){
 	}
 };
 
-Potree.Version.prototype.upTo = function(version){
+Potree.Version.prototype.up to = function(version){
```

Comment:
This is actually a syntax error and will generate an error. We should reverse this change.